test/test_sampler_multiproc.py

```diff
@@ -11,22 +12,18 @@
 from sampling_pool import ProcessPool, ThreadPool, OneProcessPool
 from mlmc.moments import Legendre
 
-def multiproces_sampler_test(hdf=False):
 
-    n_levels = 2
-    failed_fraction = 0#0.2
+def multiproces_sampler_test():
```

Comment:
Add an issue to test overhead of the multiprocesspool communication. Can be quite big for light simulations. We can possibly group more samples into single communication.